static/i18n/en.json

```diff
@@ -4,12 +4,16 @@
 		]
 	},
 	"name":"English",
-	"suggest-full-text-search": "containing '{{{SEARCH_TERMS}}}'..."
-	, "no-such-book": "No such book: {{BOOK_NAME}}"
+	"suggest-full-text-search" : "containing '{{{SEARCH_TERMS}}}'..."
+	, "no-such-book" : "No such book: {{BOOK_NAME}}"
+	, "too-many-books" : "Too many books requested ({{NB_BOOKS}}) where limit is {{LIMIT}}"
+	, "no-book-found" : "No book found"
```

Comment:
1. Isn't "No book matches selection criteria" a better message?
2. This message is not documented in qqq.json